docs/data/galexie/admin_guide/configuring.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: Configuring
+sidebar_position: 20
+---
+
+# Configuring
+
+## Steps to Configure Galexie
+
+1. **Copy the Sample Configuration**
+
+   Start with the provided sample file, [`config.example.toml`](https://github.com/stellar/go/blob/master/services/galexie/config.example.toml).
+
+2. **Rename and Update the Configuration**
+
+   Rename the file to `config.toml` and adjust settings as needed.
+
+   - **Key Settings Include:**
+
+     - **Google Cloud Storage (GCS) Bucket**
+
+       Specify the GCS bucket where Galexie will export Stellar ledger data. Update `destination_bucket_path` to the complete path of your GCS bucket, including subpaths if applicable.
+
+       ```toml
+       destination_bucket_path = "stellar-network-data/testnet"
+       ```
+
+     - **Stellar Network**
+
+       Set the Stellar network to be used in creating the data lake.
+
+       ```toml
+       network = "testnet"
+       ```
+
+     - **Data Organization (Optional)**
+
+       Configure how the exported data is organized in the GCS bucket. The example below adds 64 ledgers per file and organizes them in a directory of 1000 files.
+
+       ```toml
+        # Number of ledgers stored in each file
+        ledgers_per_file = 64
+
+        # Number of files per partition/directory
+        files_per_partition = 1000
```

Comment:
Nit, but I think what we put by defaults in docs should be what we use for our production config